components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/network/NetworkSettingsConfigurator.java

```diff
@@ -43,6 +43,10 @@ private static boolean isProxyProperty(String key) {
             || key.endsWith(".nonProxyHosts");
     }
 
+    private static boolean isTimeoutProperty(String key) {
```

Comment:
Maybe rename the method to `isConnectionProperty`?

Reply:
✅ Done